notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -382,7 +334,7 @@
         "id": "7qXFUiHLoFRw"
       },
       "source": [
-        "A service account is used to create custom training job. If you do not want to use your project's Compute Engine service account, set SERVICE_ACCOUNT to another service account ID. You can create a service account by following the [instruction](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating)."
+        "A service account is used to create custom training job. If you don't want to use your project's Compute Engine service account, set SERVICE_ACCOUNT to another service account ID. You can create a service account by following the [instruction](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating)."
```

Comment:
suggested edit:

- by following the instructions in [Create a service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating)